### ISSUE TITLE
Looking for maintainers?

### ISSUE THREAD

**Issue by mbbroberg** [phodge/homely#53]
Hey @Darksider3  & @phodge -- would you two like to continue to maintain this repo or do you want to hand it off to others? 

I ask because I gave this a go and ran into errors described in #52. That's no worries - you've given a great bit of code here that I'll see if I can piece together and keep using as a fork for a bit. I'll play around with extending it and see what happens on my branch. In the meantime, it'd be great to know how you two are feeling about it and whether you could pass over the keys to well-meaning people to keep it going. 

Thanks again for a fun project. Hope you're both holding up okay :house_with_garden:

**Comment by Darksider3** [comment]
Hi @mbbroberg,

It's not like im maintaining anything here, really. :grin: I just contributed that little piece of pacman. I'd be fine with you using that on your branch, of course. 
Im actually pretty sad, because homely seems still to be pretty helpful. At least to me, it still is.
Please let me know when you've got something ready over there. I'd like to help you out a little, in case you're still interested.  I've no keys, whatsoever, for this project or to phodge.. ^^'

Thanks for tagging me. Even though i dont really understand why. But that's nice anyhow. :)

**Comment by phodge** [comment]
Hi @mbbroberg and @Darksider3. Thanks for the interest in homely, it's nice to see other people getting some value from my many hours of work back in 2017. You guys have made me realize that unfortunately with all the things I have going on now (mostly, kids) I just don't have time to give homely the love it needs.

So, I think I'd be willing to give you access to the repo and the pypi project if actually are really keen to see this develop further. @mbbroberg: Is there anything in particular you'd like to add to homely other than just ironing out a few bugs?

**Comment by mbbroberg** [comment]
@phodge I hear you there. I'm pre-little ones and using it to pick up some new skills, Python being one of them. 

And @Darksider3, my mistake :smile:. I spot you on the new repo layout (I'm in the preview I think), and saw your note on #52.If all goes forward, I'd love to lean on you for the occasional code review if you're into it. It takes a village. 

TL;DR is exactly what you said @phodge: I'd love to nurture bug fixes and help keep homely going.

Longer thought: I'm experienced at maintaining OSS projects (see my profile if you're curious), but new to spending time deeper in the code. Thankfully I have great mentors, and I make the work part of my job by writing about it (like [this post](https://opensource.com/article/19/5/python-3-default-mac)). When I thought about the only thing I care about enough to work on, it's dotfile management. I love to fiddle with them, and I recently switched over to multiple Linux distros that are running into minor bugs. I might be inspired to work toward features after more familiarity, but I'd prefer leaning on you to weigh in before introducing scope creep.

Shoot me a note: `matt` at `opensource.com` is my gh email these days.

**Comment by mbbroberg** [comment]
Closing this since I now have access :house:
